pimd/pim_cmd.c

```diff
@@ -610,7 +610,7 @@ static void igmp_show_interfaces_single(struct pim_instance *pim,
 		if (strcmp(ifname, "detail") && strcmp(ifname, ifp->name))
 			continue;
 
-		for (ALL_LIST_ELEMENTS_RO(pim_ifp->socket_list, sock_node,
+		for (ALL_LIST_ELEMENTS_RO(pim_ifp->gm_socket_list, sock_node,
```

Comment:
Can you let us know if you have something in your mind for these changes?

Reply:
I've already noted in the PR comment, the names are too confusing. `pim_interface` contains a lot of bits from both PIM and IGMP. It is absolutely not clear that `socket_list` refers to only IGMP sockets and not PIM.